docs/rules.md

```diff
@@ -0,0 +1,40 @@
+# Available Rules
```

Comment:
I think the url for this file has changed.
I think we should probably change it to `docs/rules/index.md`.

Reply:
Changed! :raised_hands: